src/tools/rbd/action/BenchWrite.cc

```diff
@@ -262,6 +262,11 @@ int execute(const po::variables_map &vm) {
   } else {
     bench_io_size = 4096;
   }
+  if ((bench_io_size > 100*1024*1024) || (bench_io_size < 1))
```

Comment:
You are receiving a floating point exception when allocating memory?  or is this a different exception?  What type of system are you running rbd against where it fails to allocate a single 1GB buffer at the start of the test?